### ISSUE TITLE
SEO 해결하기.

### ISSUE THREAD

**Issue by kimalec** [WizardFactory/BlogSyncer#240]
동적으로 생성되는 페이지들은 SEO에 검색되지 않는 문제가 있다. 그 문제를 아래 같은 서비스나 대처 방안을 모색해야 한다.

참고 : https://prerender.io

**Comment by kimalec** [comment]
koa-prerender
At RisingStack, we do not like half measures, and we wanted to fix this - this is when prerender.io came into the picture. It is an external service (also an open source project) that renders your site on an external server with a headless browser and sends the result back in HTML. 
It makes your site readable for most search engines but also breaks your AngularJS bindings so you cannot use it for real human users.
Because our site uses Koa, the generator based Node.js framework which is not supported by prerender.io, we had to implement it ourself. 
So RisingStack released a koa-prerender middleware for Koa. 
In a nutshell: it detects the crawlers from the request parameters (_escaped_fragment_, user-agent etc.), then calls the external prerender service and responds with the static HTML content.
We were happy because our site was finally reachable for most search engines like Google and Yahoo but still not for all. Also the user-agents can change, and we do not want to maintain it. We kept looking for a better solution.

http://blog.risingstack.com/from-angularjs-to-react-the-isomorphic-way/
